objects/Condor.py

```diff
@@ -0,0 +1,55 @@
+import os
+import fileinput
```

Comment:
sick! never seen this `fileinput` package - imma use that! But.....see above about the template idea for the sub file. Might get sticky with tracking real IP addresses

Reply:
yeah it's kind of nice! and yeah the sub file will be changed back.